pkg/util/fsnotification/fsnotification.go

```diff
@@ -0,0 +1,52 @@
+package fsnotification
```

Comment:
will add doc.go.  open to suggestions on the name, but calling it fsnotify seemed worse, i think fsnotification was one of clayton's suggestions.